apps/pkcs12.c

```diff
@@ -85,38 +86,41 @@ const OPTIONS pkcs12_options[] = {
      "Do not load certificates from the default certificates store"},
 
     OPT_SECTION("Input"),
-    {"inkey", OPT_INKEY, 's', "Private key if not infile"},
-    {"certfile", OPT_CERTFILE, '<', "Load certs from file"},
+    {"inkey", OPT_INKEY, 's', "Private key if not in input file"},
```

Comment:
-in input file ?

Reply:
Do you mean `if not in -in input file`?

Reply:
I think the existing text in the PR is clear enough.

Reply:
Having a closer look at the code again I found none of these variants was correct
because the input file given with -in is just the default for the key if -inkey is not given.
So I've just corrected this to "Private key, else read from -in input file".